libraries/abstractions/secure_sockets/test/iot_test_tcp.c

```diff
@@ -440,6 +440,11 @@ size_t xHeapA;
 /* Group declaration required by Unity framework. */
 TEST_GROUP( Full_TCP );
 
+/* Extended test group for tests which aren't required for qualification
+ * but demonstrate additional features or preferred behavior.
+ */
+TEST_GROUP( Full_TCP_Extended );
```

Comment:
Should this be added to [aws_test_runner.c](https://github.com/aws/amazon-freertos/blob/master/tests/common/aws_test_runner.c#L82) as well? I am not sure what is required for release and thus I might be wrong.
If so, we need to add #define in [aws_test_runner_config.h](https://github.com/aws/amazon-freertos/blob/master/vendors/pc/boards/windows/aws_tests/config_files/aws_test_runner_config.h#L56) in vendor directory of all respective vendors.

Reply:
Thanks @AniruddhaKanhere, we will not add the extended tests here.  These are optional tests for now.